src/Image.js

```diff
@@ -29,19 +30,28 @@ export default class Image extends React.Component<ImageProps, ImageState> {
 
     static defaultProps = {
         transitionDuration: 300,
-        tint: "dark"
+        tint: "dark",
+        onError: () => {}
     };
 
     state = {
         uri: undefined,
         intensity: new Animated.Value(100)
     };
 
-    async load({uri, options = {}}: ImageProps): Promise<void> {
+    async load({uri, options = {}, onError}: ImageProps): Promise<void> {
         if (uri) {
-            const path = await CacheManager.get(uri, options).getPath();
-            if (this.mounted) {
-                this.setState({ uri: path });
+            try {
+                const path = await CacheManager.get(uri, options).getPath();
+                if (this.mounted) {
+                    if (path) {
+                        this.setState({ uri: path });
+                    } else {
+                        onError(new Error("Could not load image"));
```

Comment:
Here the error format should with symetric with onError from `<Image>` no?

Reply:
I updated it to use the same format